### PR TITLE
Re-add TVR Parlemen subchannels

### PR DESCRIPTION
**Pull request by Hans5958**
Not sure why the subchannels are removed, even though `[Not 24/7]` is there last time. Let's try again with this PR.

### CODE REVIEW

File: streams/id.m3u [hunk 1]
```diff
@@ -553,6 +553,20 @@ http://103.18.181.69:1935/golive/livestream7/playlist.m3u8
 http://103.18.181.69:1935/golive/livestream10/playlist.m3u8
 #EXTINF:-1 tvg-id="TVRParlemenKomisiXI.id" status="error",TVR Parlemen Komisi XI (720p) [Not 24/7]
 http://103.18.181.69:1935/golive/livestream11/playlist.m3u8
+#EXTINF:-1 tvg-id="TVRParlemenKomisiVI.id",TVR Parlemen Komisi VI [Not 24/7]
+http://103.18.181.69:1935/golive/livestream6/playlist.m3u8
+#EXTINF:-1 tvg-id="TVRParlemenKomisiVIII.id",TVR Parlemen Komisi VIII [Not 24/7]
+http://103.18.181.69:1935/golive/livestream8/playlist.m3u8
+#EXTINF:-1 tvg-id="TVRParlemenBadanLegislatif.id"",TVR Parlemen Badan Legislatif [Not 24/7]
```
Comment (UltraHDR) [inline streams/id.m3u:560]:
```suggestion
#EXTINF:-1 tvg-id="TVRParlemenBadanLegislatif.id",TVR Parlemen Badan Legislatif [Not 24/7]
```

**Comment by UltraHDR** [comment]
Links don't load. Maybe they are [Geo-blocked] ?

**Comment by Hans5958** [comment]
> Links don't load. Maybe they are [Geo-blocked] ?

Well, I mean it is not online 24/7, so that is expected. It is not geo-blocked, you could try the `TVRParlemen.id` one if you want to try if it is blocked. https://streamtest.in/ said it works.

**Comment by UltraHDR** [comment]
> > Links don't load. Maybe they are [Geo-blocked] ?
> 
> Well, I mean it is not online 24/7, so that is expected. It is not geo-blocked, you could try the `TVRParlemen.id` one if you want to try if it is blocked. https://streamtest.in/ said it works.

http://103.18.181.69:1935/golive/livestream6/playlist.m3u8 worked. I'll approve

**Review by UltraHDR — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**